kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3OutputStream.java

```diff
@@ -164,7 +171,7 @@ private ObjectMetadata newObjectMetadata() {
     return meta;
   }
 
-  private MultipartUpload newMultipartUpload() throws IOException {
+  protected MultipartUpload newMultipartUpload() throws IOException {
```

Comment:
In first place I had an idea of making it accessible for testing, then forgot to change back.